---a/net/minecraft/world/biome/BiomeGenMutated.java
+++b/net/minecraft/world/biome/BiomeGenMutated.java
@@ -1,20 +1,19 @@
 package net.minecraft.world.biome;
 
-import com.google.common.collect.Lists;
 import java.util.Random;
+
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.World;
 import net.minecraft.world.chunk.ChunkPrimer;
 import net.minecraft.world.gen.feature.WorldGenAbstractTree;
-import net.minecraftforge.fml.relauncher.Side;
-import net.minecraftforge.fml.relauncher.SideOnly;
 
-public class BiomeGenMutated extends BiomeGenBase
-{
+import com.google.common.collect.Lists;
+
+public class BiomeGenMutated extends BiomeGenBase {
+
     protected BiomeGenBase baseBiome;
     private static final String __OBFID = "CL_00000178";
 
-    public BiomeGenMutated(int p_i45381_1_, BiomeGenBase p_i45381_2_)
-    {
+    public BiomeGenMutated(int p_i45381_1_, BiomeGenBase p_i45381_2_) {
         super(p_i45381_1_);
         this.baseBiome = p_i45381_2_;
         this.func_150557_a(p_i45381_2_.color, true);
@@ -40,51 +40,31 @@
         this.maxHeight = p_i45381_2_.maxHeight + 0.2F;
     }
 
-    public void decorate(World worldIn, Random p_180624_2_, BlockPos p_180624_3_)
-    {
+    public void decorate(World worldIn, Random p_180624_2_, BlockPos p_180624_3_) {
         this.baseBiome.theBiomeDecorator.decorate(worldIn, p_180624_2_, this, p_180624_3_);
     }
 
-    public void genTerrainBlocks(World worldIn, Random p_180622_2_, ChunkPrimer p_180622_3_, int p_180622_4_, int p_180622_5_, double p_180622_6_)
-    {
+    public void genTerrainBlocks(World worldIn, Random p_180622_2_, ChunkPrimer p_180622_3_, int p_180622_4_, int p_180622_5_, double p_180622_6_) {
         this.baseBiome.genTerrainBlocks(worldIn, p_180622_2_, p_180622_3_, p_180622_4_, p_180622_5_, p_180622_6_);
     }
 
-    public float getSpawningChance()
-    {
+    public float getSpawningChance() {
         return this.baseBiome.getSpawningChance();
     }
 
-    public WorldGenAbstractTree genBigTreeChance(Random p_150567_1_)
-    {
+    public WorldGenAbstractTree genBigTreeChance(Random p_150567_1_) {
         return this.baseBiome.genBigTreeChance(p_150567_1_);
     }
 
-    @SideOnly(Side.CLIENT)
-    public int getFoliageColorAtPos(BlockPos p_180625_1_)
-    {
-        return this.baseBiome.getFoliageColorAtPos(p_180625_1_);
-    }
-
-    @SideOnly(Side.CLIENT)
-    public int getGrassColorAtPos(BlockPos p_180627_1_)
-    {
-        return this.baseBiome.getGrassColorAtPos(p_180627_1_);
-    }
-
-    public Class getBiomeClass()
-    {
+    public Class getBiomeClass() {
         return this.baseBiome.getBiomeClass();
     }
 
-    public boolean isEqualTo(BiomeGenBase p_150569_1_)
-    {
+    public boolean isEqualTo(BiomeGenBase p_150569_1_) {
         return this.baseBiome.isEqualTo(p_150569_1_);
     }
 
-    public BiomeGenBase.TempCategory getTempCategory()
-    {
+    public BiomeGenBase.TempCategory getTempCategory() {
         return this.baseBiome.getTempCategory();
     }
-}
+}
